model/host/hostutil.go

```diff
@@ -27,8 +29,13 @@ import (
 	"go.mongodb.org/mongo-driver/bson"
 )
 
+func init() {
```

Comment:
Since this package imports the distro package which reseeds it, I think this is unecessary

Reply:
Removed.